msvc/vsimporter-templates/WinStore10-App/Package-native.appxmanifest

```diff
@@ -8,7 +8,7 @@
 
   <Identity
     Name="$guid9$"
-    Publisher="islandwood"
+    Publisher="CN=islandwood"
```

Comment:
>CN=islandwood [](start = 15, length = 13)

I'm nervous about backwards compat.  Does anyone have a machine without 14393?
How can we validate we won't break external developers who haven't installed 14393 SDK.